ddtrace/context.py

```diff
@@ -123,11 +130,74 @@ def sampling_priority(self, value):
     @property
     def _traceparent(self):
```

Comment:
not a huge fan of having these w3c concepts mixed with the `Context` we're implicitly coupling the implementations which will make it hard to refactor one, the other or both in the future.

An alternative might be to just make these standalone functions that take a "`Context`-like" object and operate on that

Reply:
@Kyle-Verhoog  would this be a blocker?

Reply:
I agree we should reconsider the coupling of w3c headers and the context object but would suggest that be done as a follow-up refactoring to this change.

Reply:
:+1: i'm ok to not block on it